src/main/java/com/minecolonies/coremod/entity/ai/citizen/baker/BakerRecipes.java

```diff
@@ -76,11 +70,11 @@
 
         final List<ItemStack> inputBread = new ArrayList<>();
         inputBread.add(new ItemStack(Items.WHEAT, REQUIRED_WHEAT));
-
+        final StandardFactoryController sfc = StandardFactoryController.getInstance();
         recipes = new ImmutableList.Builder<RecipeStorage>()
```

Comment:
Use IRecipeStorage
There is no need for the actuall type here. The Interface needs to suffice.